webapp/lib/app/configurator/view.dart

```diff
@@ -24,6 +24,24 @@ class ConfigurationPageView extends PageView {
   ButtonElement saveConfigurationButton;
   SpanElement saveStatusElement;
 
+  bool _unsavedChanges = false;
+
+  bool get unsavedChanges => _unsavedChanges;
+  set unsavedChanges(bool status) {
+    if (status == _unsavedChanges) {
+      return;
+    }
+
+    _unsavedChanges = status;
+    if (status) {
```

Comment:
I think this makes it slightly easier to interpret what the if is about:
```suggestion
    if (_unsavedChanges) {
```